src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -740,6 +741,20 @@ public Object getObject(int index) throws SQLServerException {
             returnValue = getTime(index);
         } else if (type == java.sql.Timestamp.class) {
             returnValue = getTimestamp(index);
+        } else if (type == java.time.LocalDateTime.class || type == java.time.LocalDate.class
+                || type == java.time.LocalTime.class) {
+            java.time.LocalDateTime ldt = getLocalDateTime(index);
+            if (null == ldt) {
+                returnValue = null;
+            } else {
+                if (type == java.time.LocalDateTime.class) {
+                    returnValue = ldt;
+                } else if (type == java.time.LocalDate.class) {
+                    returnValue = ldt.toLocalDate();
+                } else {
+                    returnValue = ldt.toLocalTime();
+                }
+            }
```

Comment:
While you're at it, could you also copy over the code for OffsetDateTime and OffsetTime from SQLServerResultSet?